### ISSUE TITLE
False-positive for anonymous functions

### ISSUE THREAD

**Issue by r2k1** [tomarrell/wrapcheck#17]
Wrapcheck throws an error for the next case:

```go
func Test_AnonFunc(t *testing.T) {
	test := func() error {
		return errors.New("test")
	}
	fmt.Println(test())
}
```

`error returned from external package is unwrapped: sig: func errors.New(text string) error`

**Comment by tomarrell** [comment]
Thanks for the report. Will have a look when I get some time.
